source/_reference-g-s.html.md.erb

```diff
@@ -5336,6 +5357,31 @@ Represents individual user settings for a specific sheet.  User settings may be
 **id** | number | Id of the report, sheet, Sight (aka dashboard), or template from which the enclosing report, sheet, Sight, or template was created
 **type** | string | **report**, **sheet**, **sight**, or **template**
 
+### SummaryField Object
+
+     |     |     |
+-----|-----|-----|
+**id** | number | SummaryField Id
+**contactOptions** | [ContactOption](#contactoption-object)[] | Array of ContactOption objects to specify a pre-defined list of values for the column. Column **type** must be **CONTACT_LIST** 
+**createdAt** | timestamp | Time of creation
+**createdBy** | [User](#user-object) | User object containing name and email of the summaryField's author
+**displayValue** | string | Visual representation of cell contents, as presented to the user in the UI.  See [Cell Reference](#cell-reference).
+**format** | string | The format descriptor (see [Formatting](#formatting))<br/>Only returned if the **include** query string parameter contains **format** and this column has a non-default format applied to it.
+**formula** | string | The  formula for a cell, if set. NOTE: calculation errors or problems with a formula do not cause the API call to return an error code. Instead, the response contains the same value as in the UI, such as cell.value = "#CIRCULAR REFERENCE".
+**hyperlink** | [Hyperlink](#hyperlink-object) | A hyperlink to a URL, sheet, or report
+**image** | [Image](#image-object) | The image that the cell contains.<br/>Only returned if the cell contains an image.
+**index** | number | Column index or position. This number is zero-based. 
+**locked** | Boolean | Indicates whether the column is locked. In a response, a value of **true** indicates that the column has been locked by the sheet owner or the admin.
+**lockedForUser** | Boolean | Indicates whether the column is locked for the requesting user. This attribute may be present in a response, but cannot be specified in a request.
+**modifiedAt** | timestamp | Time of last modification
+**modifiedBy** | [User](#user-object) | User object containing name and email of the summaryField's author
+**objectValue** | [ObjectValue](#objectvalue-object) | Required for date and contact fields
+**options** | string[] | When applicable for **PICKLIST** column type. Array of the options available for the column
```

Comment:
:thinking: I don't know the standard to referring to "column type" for a field, but I assume we're still calling them "column types".. that said, "available for the column" should probably be "available for the field"

Reply:
@TaranChristensen @kfansler We are implying in the docs that all supported column types are ok as field types, but by observation that doesn't appear to be correct. Looks like it might only be TEXT_NUMBER, PICKLIST, DATE, CONTACT_LIST, CHECKBOX. Taran, can you confirm? If that is the case we should treat the field.type separately from column.type.

Reply:
Actually, the only type that /isn't/ possible-- that is to say, the only type that doesn't overlap-- is MultiContact. And, the upcoming MultiPicklist. Every other type should be supported in both cells and fields.

Reply:
Taran, I'm confused. How do I get to say a PREDECESSOR_LIST through the UI: 
<img width="365" alt="Screen Shot 2019-08-12 at 1 52 04 PM" src="https://user-images.githubusercontent.com/10661345/62898008-5599aa00-bd09-11e9-9bdc-f62e4582a974.png">

Reply:
I'm so sorry, I'm not being very reliable here. Of the "main" column types, multi-picklist is the only that Sheet Summary doesn't support. But you're right, there are a handful of others that you can create in various ways for a sheet as a column that you can not create for a field-- Duration, Predecessor, the various system columns.. Maybe you're correct that it is a separate subset of types. I'm weary that supporting both types will be more difficult than necessary, but in the public API maybe it's better to be explicit and worry less about repetition.

Reply:
Ok, gotcha. We are on the same page.